### PR TITLE
docs(readme): fix a typo

### PR DESCRIPTION
**Pull request by helloausrine**
Hi,

Highlighted the project's name and fixed a spelling mistake.

Great project! Peace!

### CODE REVIEW

**Review by kamranayub — COMMENTED** [review]
Thanks for the typo fix. Not sure about the bolding as I don't know if that adds a lot to the copy... @eonarheim or @jedeen what do you think?

I'm fine approving the typo though. 👍

**Comment by eonarheim** [comment]
@helloausrine thanks for the typo fix, please undo the bolding and we'll merge 👍

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ![Sweep Stacks](http://excaliburjs.com/assets/images/homepage-xp.png)
 
-Excalibur is a free game engine written in TypeScript for making 2D games in HTML5 canvas. Our goal is to make it easier for you to create 2D HTML/JS games, whether you're new to game development or you're an experienced game developer. We take care of all of the boilerplate engine code, cross-platform targeting (using [browserstack](http://browserstack.com/) 😎), and more! Use as much or as little as you need!
+**Excalibur** is a free game engine written in TypeScript for making 2D games in HTML5 canvas. Our goal is to make it easier for you to create 2D HTML/JS games, whether you're new to game development or you're an experienced game developer. We take care of all of the boilerplate engine code, cross-platform targeting (using [browserstack](http://browserstack.com/) 😎), and more! Use as much or as little as you need!
```
Comment (eonarheim) [inline README.md:12]: Remove bolding

**Review by eonarheim — CHANGES_REQUESTED** [review]
@helloausrine thanks! Remove the bolding change, and we will merge

File: README.md (hunk 1 above)
Comment (helloausrine) [reply README.md]: Bolding removed.

**Pull request merged**